xbmc/interfaces/python/typemaps/python.buffer.intm

```diff
@@ -33,4 +33,6 @@
       ${api}.flip(); // prepare the buffer for reading from
     }
     else
-      throw XBMCAddon::WrongTypeException("argument \"%s\" for \"%s\" must be a string, bytes or a bytearray", "${api}", "${method.@name}");
\ No newline at end of file
+    {
+      throw XBMCAddon::WrongTypeException("argument \"%s\" for \"%s\" must be a string, bytes or a bytearray", "${api}", "${method.@name}");
+    }
```

Comment:
Maybe while at it add an empty space at the end of the file?

Reply:
Because it's swig insert stuff, I'll leave it as is. It really is difficult for me to predict changes easily in the swig template stuff